packages/medusa/src/api/routes/store/auth/index.ts

```diff
@@ -17,3 +17,12 @@ export default (app) => {
 
   return app
 }
+
```

Comment:
add `export class StoreGetAuthRes {
    customer: Customer
}`

Reply:
I went with `StoreAuthRes` as it made a tiny bit more sense + to stay a bit consistent with the naming conventions throughout the codebase. What do you think?

Reply:
@zakariaelas I agree makes perfect sense :)